mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -190,11 +178,9 @@ function PoolPairRowYour ({
   const [symbolA, symbolB] = (pair?.tokenA != null && pair?.tokenB != null)
     ? [pair.tokenA.displaySymbol, pair.tokenB.displaySymbol]
     : data.symbol.split('-')
-  const IconA = getNativeIcon(symbolA)
-  const IconB = getNativeIcon(symbolB)
   const toRemove = new BigNumber(1).times(data.amount).decimalPlaces(8, BigNumber.ROUND_DOWN)
   const ratioToTotal = toRemove.div(pair?.totalLiquidity?.token ?? 1)
-  const symbol = `${symbolA}-${symbolB}`
+  const symbol = `${symbolA} – ${symbolB}`
```

Comment:
Let's remove the spacing in between. To make it consistent with DeFiScan

Reply:
updated